### DISCUSSION TITLE
Mount Arena middleware on route

### DISCUSSION THREAD

**Discussion by andychongyz** [blitz-js/blitz#1401] [Q&A] [answered]
Hi, I'm new to Blitz and Next js.

Anyone know how can I mount https://github.com/bee-queue/arena on the express router? Since Blitz build on Next, which rely on Express 😅

**Comment by theZieger** [comment, marked as answer]
Hi @andychongyz you may want to watch this issue: https://github.com/blitz-js/blitz/issues/303 :slightly_smiling_face:

**Comment by andychongyz** [comment]
Thanks @theZieger , I took a look at https://nextjs.org/docs/advanced-features/custom-server, the `handle` function in the else statement will delegate the unmatched route to the default server from blitz/next?

**Reply by theZieger** [reply]
Not really sure. Maybe one of the maintainers can help with this question. About the custom server I only know because I searched how to implement socket.io directly with Blitz.
